core/src/integration-test/java/com/scalar/db/storage/StorageMultipleClusteringKeyScanIntegrationTestBase.java

```diff
@@ -41,14 +36,13 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
-import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.Test;
 
 @SuppressFBWarnings(
     value = {"MS_CANNOT_BE_FINAL", "MS_PKGPROTECT", "ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD"})
-public abstract class StorageMultipleClusteringKeysIntegrationTestBase {
+public abstract class StorageMultipleClusteringKeyScanIntegrationTestBase {
```

Comment:
I renamed it from `StorageMultipleClusteringKeysIntegrationTestBase` to `StorageMultipleClusteringKeyScanIntegrationTestBase`. I also renamed the classes that extends this class in the same way.